u_parrots.go

```diff
@@ -495,6 +495,25 @@ func (uconn *UConn) generateRandomizedSpec(WithALPN bool) (ClientHelloSpec, erro
 	if err != nil {
 		return p, err
 	}
+
+	if tossBiasedCoin(0.4) {
+		p.TLSVersMin = VersionTLS10
+		p.TLSVersMax = VersionTLS13
```

Comment:
Is VersionTLS13 == VersionTLS12?

Reply:
no

Reply:
What version ends up in the handshake version field then? It seems like you'll be sending 0x0304 but you want to send 0x0303, or is that logic somewhere else?

Reply:
that logic is somewhere else, those fields are to store real versions and generate all the things accordingly